src/components/Map/BoothList.tsx

```diff
@@ -0,0 +1,136 @@
+import { NavLink } from 'react-router-dom';
+
+import { styled } from 'styled-components';
+
+import InfoWithIcon from '../MapDetail/InfoWithIcon';
+import Booth from '../../types/Booth';
+
+const Container = styled.div`
+  overflow-y: scroll;
+  width: 100%;
+  height: 80%;
+  margin-bottom: 100px;
+`;
+
+const PreButton = styled(NavLink)`
+  cursor: pointer;
+  text-decoration: none;
+  color: inherit;
+`;
+
+const BoothContainer = styled.div`
+    flex: 1;
+    display: flex;
+    align-items: center;
+    padding: 10px;
+    width: 100%;
+    justify-content: space-between;
+    border-bottom: 1px solid #CEDCEA;
+
+    img{
+        width: 100px;
+        height: 100px;
+    }
+`;
+
+const BoothContent = styled.div`
+    width: 35%;
+    height: 100px;
+    margin-left: 10px;
+    margin-right: 10px;
+    
+    p {
+        font-size: 17px;
+        margin-top: 6px;
+        margin-bottom: 6px;
+        color: #0047C9;
+        overflow: hidden;
+    }
+`;
+
+const BoothInfoBottom = styled.div`
+    color: #BBC7D3;
+    font-size: 16px;
+
+    div {
+      display: inline-block;
+      margin-top: 2.5px;
+      margin-bottom: 2.5px;
+      margin-right: 5px;
+    }
+`;
+
+const LikedNumber = styled.div` 
+    padding-top: 2.5px;
+    padding-bottom: 2.5px;
+    padding-left: 10px;
+    padding-right: 10px;
+    display: flex;
+    background-color: #EBF2FF;
+    border-radius: 15px;
+    align-items: center;
+    justify-content: center;
+    color: #0047C9;
+  
+    span {
+      font-size: 13px;
+      margin: 1px;
+    }
+`;
+
+type BoothListProps = {
+  booths: Booth[];
+  selectedDay: string;
+};
+
+export default function BoothList({
+  booths,
+  selectedDay,
+}: BoothListProps) {
+  if (!booths) { return []; }
```

Comment:
사용한다면 이렇게 되어야 할 것 같네요~
```suggestion
  if (!booths) { 
  return null; 
}
```